diffpy/snmf/stretchednmfapp.py

```diff
@@ -2,6 +2,7 @@
 import argparse
 
 from diffpy.snmf.io import load_input_signals, initialize_variables
+from diffpy.snmf.subroutines import *
```

Comment:
don't ever do this.  Import what you want only.  This is a future headache waiting to happen.